### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by dannyprok**
Updated scaffold and project README.md files
Removed WindowsControl.scala - not referenced and should probably live elsewhere.
Upgraded Geckodriver version to 0.21.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,91 +1,84 @@
...
+- **cucumber** -> [OPTIONAL] A Boolean property which defaults to `false` (Scalatest).  Selecting `true` will provide you with all the required dependencies to run Cucumber, as well as example StepDef and TestRunner objects.
 
-    drivers/installGeckodriver.sh
-    drivers/installChromedriver.sh
-
-
-These scripts use sudo to get the right permissions so you will likely be prompted to enter your password.
-
-**Note** you will need up to date versions of Firefox and Chrome installed on your device to be able to use these drivers.
+Upon Answering these questions will result in a blank test suite with all the basic requirements needed for ui tests in your current directory.
```
Comment (ejayaraman) [inline README.md]: Should this be `result in an example test suite` instead of `blank test suite`

File: README.md [hunk 2]
```diff
@@ -98,45 +91,73 @@ Alternatively if you access [www.browserstack.com/automate](http://www.browserst
...
+### Running the ui-test-template.g8 tests
+There are test scripts (written in bash) in the `tests/` folder which run UI tests against serveral combinations of browser->test-runner->test-environment.  To successfully run the tests you will need to satisfy the following pre-requisites: 
 
-Now run the **run_browserstack.sh** script.
+- [Install Giterate CLI](#install-giterate)
+- Install [Docker]()
+- Build the latest HMRC Digital Chrome and Firefox images (see Confluence)
+- Install chromedriver and geckodriver (reference the project [README.md](./src/main/g8/README.md) )
+- Install and configure [Service Manager](https://github.com/hmrc/service-manager) (see Confluence)
+- Install [Mongo](https://docs.mongodb.com/manual/installation/)
```
Comment (ejayaraman) [inline README.md]: It is better to refer to Confluence for Mongo Installation instead of mongo docs.

File: src/main/g8/README.md [hunk 3]
```diff
@@ -1,69 +1,52 @@
...
+Run the following command to start services locally:
 
     sudo mongod
-    sm --start ACCEPTANCE_TEST_TEMPLATE -f
-    ./run_local.sh
-
-### How to use this template
-
-We _always_ keep UI driven tests to a minimum, preferring instead to drive tests down into Unit and Integration layers. Please read the MDTP Test Approach for more details.
+    sm --start UI_TEST_TEMPLATE
```
Comment (ejayaraman) [inline src/main/g8/README.md]: Would it make sense to append -f option. `sm --start UI_TEST_TEMPLATE -f`

File: README.md [hunk 4]
```diff
@@ -98,45 +91,73 @@ Alternatively if you access [www.browserstack.com/automate](http://www.browserst
...
-For Example:
+## Development
+If you'd like to contribute to the ui-test-template you'll need to test your changes before raising a PR.  
 
-    BS_Win8_Chrome_v64
+### Manually generating a UI Test project from you local changes
+To create a test project from your local changes, execute the following command from the parent directory of your local copy of the template:
 
-Once the JSON objects have been created, these need to be added to the run_browserstack.sh script.
+    g8 file://ui-test-template.g8/ --name=my-test-project --cucumber=true
```
Comment (ejayaraman) [inline README.md]: I can see test-ui-test-template.sh is creating a test repo to test the changes. So is the above step still required?

Comment (dannyprok) [reply README.md]: It isn't clear what this step is for, I'll have a go at re-wording it...  

If you'd like to contribute to the template, the best way to do so (that I know of) is to make the changes locally, then generate and run the test project manually to ensure your changes haven't broken anything.  the test script is not exhaustive and would require some work to increase it's coverage (Proxy tests, browserstack tests...etc).  

So, running test-ui-test-template.sh should not take the place of manually testing your changes.

Comment (dannyprok) [reply README.md]: renamed section heading

File: README.md [hunk 5]
```diff
@@ -98,45 +90,73 @@ Alternatively if you access [www.browserstack.com/automate](http://www.browserst
...
+- Install and configure [Service Manager](https://github.com/hmrc/service-manager) (see Confluence)
+- Install Mongo (see Confluence)
 
->**Note:** if you only wish to run either the browsers or devices you need to remove the relevant entry from within run_browserstack.sh
+Copy `tests/ui-test-template-tests.sh` to the parent directory of your local copy of the ui-test-template.g8 project.  Execute the script without params:
 
->**Note:** the changes made to browserConfig.properties should not be pushed to GitHub and therefore you should make sure that this file is included on the `.gitignore` file for your project.
+    ./ui-test-template-tests.sh
 
+**Note:** Currently these tests do not assert that the test run was successful.
```
Comment (chrisjameswright) [inline README.md]: Should we be clearer about what should or shouldn't happen?

**Comment by craigtd** [comment]
👍

**Pull request merged**